egs/aishell/s10/chain/tdnnf_layer.py

```diff
@@ -116,7 +108,7 @@ def __init__(self, big_dim, small_dim):
         self.batchnorm1 = nn.BatchNorm1d(num_features=big_dim)
         self.linear = OrthonormalLinear(dim=big_dim,
                                         bottleneck_dim=small_dim,
-                                        time_stride=0)
+                                        kernel_size=1)
         self.batchnorm2 = nn.BatchNorm1d(num_features=small_dim)
 
     def forward(self, x):
```

Comment:
I'm surprised you didn't implement the TDNN_F layer in the "obvious" way with 1-d convolution.

Reply:
Inside `OrthonormalLinear`, it is `nn.Conv1d`. So it is ineeded implemented via 1-d convolution.

Reply:
kernel_size=1 doesn't look right.  Some extremely weird stuff is going on in this PR.

Reply:
Hi, dan, this code's behavior is different with before code's. just changed the param. the original stride version use kernel_size=1 as default also

Reply:
It's getting less clear to me, not more clear, and  in any case the code is not right. 
You showed various experimental numbers but you never said with any clarity what code or parameters each one corresponded to.
I would prefer if you just went back to the original code and made the small and very specific changes that I asked for.

Reply:
sorry about this